src/compiler/diagnosticMessages.json

```
@@ -3028,6 +3028,10 @@
         "category": "Error",
         "code": 2792
     },
+    "Template literal type argument '{0}' is not literal type or a generic type.": {

```

Just noticed this when handling a merge conflict:

```suggestion
    "Template literal type argument '{0}' is not a literal type or a generic type.": {
```